src/ledger/LedgerManagerImpl.cpp

```diff
@@ -969,6 +971,26 @@ LedgerManagerImpl::processFeesSeqNums(std::vector<TransactionFramePtr>& txs,
     }
 }
 
+void
+LedgerManagerImpl::prefetchTransactionData(
+    std::vector<TransactionFramePtr>& txs)
+{
+    auto& root = mApp.getLedgerTxnRoot();
+    for (auto const& tx : txs)
+    {
+        stellar::prefetchAccount(root, tx->getSourceID());
+        auto const& ops = tx->getOperations();
+        for (auto const& op : ops)
```

Comment:
I think we could extract method in TransactionFrame to get all of used source ID where?

for (auto const& account : tx->getSourceAccounts()) ....

Reply:
I'm not sure this would be useful. We have this loop used only once, and it seems sufficient to simply utilize the public interface of `TransactionFrame` to get accounts.